src/Protocol/Activity/ActivityNamespace.php

```diff
@@ -5,7 +5,7 @@
 /**
  * Activity namespaces constants
  */
-final class ANamespace
+final class ActivityNamespace
```

Comment:
Why not just `Friendica\Protocol\ActivityNamespace`? is there anything else in the `Friendica\Protocol\Activity` namespace?

Reply:
Yes, the `ObjectType` class

Reply:
tbh I thought @annando doesn't like such long namespaces... At first I had `Friendica\Protocol\Activity\NS`.. but this shortname must not be used ...